library/src/main/java/com/wultra/android/mtokensdk/api/operation/model/UserOperation.kt

```diff
@@ -159,4 +168,33 @@ data class FormData(
          * Note that attributes can be presented with different classes (Starting with Attribute*) based on the attribute type.
          */
         @SerializedName("attributes")
-        val attributes: List<Attribute>)
\ No newline at end of file
+        val attributes: List<Attribute>)
+
+data class OperationUIData(
+        /**
+         * Order of the buttons
```

Comment:
This comment is now enough - use the one already in iOS which is much better

Reply:
👍